VORP-Housing[Client-Server]/vorphousing_sv/vorp_housing_sv_init.cs

```diff
@@ -21,107 +20,134 @@ public vorp_housing_sv_init()
+                int charIdentifier = UserCharacter.charIdentifier;
+
+                PlayerList PL = Players;
+                Player _source = PL[source];
+                string sid = "steam:" + _source.Identifiers["steam"];
 
-                    PlayerList PL = new PlayerList();
-                    Player _source = PL[source];
-                    string sid = "steam:" + _source.Identifiers["steam"];
+                dynamic tableExist = await Exports["ghmattimysql"].executeSync("SELECT * FROM information_schema.tables WHERE table_schema = 'vorpv2' AND table_name = 'rooms' LIMIT 1;", new string[] { });
+                if (tableExist.Count == 0)
+                {
+                    Logger.Error("Server.Init.GetRooms(): SQL table \"rooms\" doesn't exist");
+                    return;
+                }
 
-                    dynamic result = await Exports["ghmattimysql"].executeSync("SELECT * FROM rooms WHERE identifier = ? AND charidentifier = ?", new object[] { sid, charIdentifier });
+                dynamic result = await Exports["ghmattimysql"].executeSync("SELECT * FROM rooms WHERE identifier = ? AND charidentifier = ?", new object[] { sid, charIdentifier });
+                Logger.Debug(JsonConvert.SerializeObject(result));
 
-                    Dictionary<int, Room> _Rooms = Rooms.ToDictionary(h => h.Key, h => h.Value);
+                Dictionary<int, Room> _Rooms = Rooms.ToDictionary(h => h.Key, h => h.Value);
 
-                    if (result.Count != 0)
+                if (result != null && result.Count > 0)
+                {
+                    foreach (var r in result)
                     {
-                        foreach (var r in result)
-                        {
-                            int roomId = r.interiorId;
-                            string identifier = r.identifier;
-                            int charidentifier = r.charidentifier;
-                            _Rooms[roomId].Identifier = identifier;
-                            _Rooms[roomId].CharIdentifier = charidentifier;
+                        int roomId = r.interiorId;
+                        string identifier = r.identifier;
+                        int charidentifier = r.charidentifier;
+                        _Rooms[roomId].Identifier = identifier;
+                        _Rooms[roomId].CharIdentifier = charidentifier;
 
-                        }
-                        string rooms = JsonConvert.SerializeObject(_Rooms);
-                        cb(rooms);
-                    }
-                    else
-                    {
-                        string rooms = JsonConvert.SerializeObject(_Rooms);
-                        cb(rooms);
                     }
+                    string rooms = JsonConvert.SerializeObject(_Rooms);
+                    TriggerClientEvent("vorp_housing:ListRooms", rooms);
+                    //cb(rooms);
                 }
-                catch (Exception e)
+                else
```

Comment:
Is there a reason for the else, as `_Rooms` is made before the logic and is updated if successful.